configs/swin_transformer/swin-tiny_16xb64_in1k_oss.py

```diff
@@ -0,0 +1,83 @@
+_base_ = [
```

Comment:
Please remove this file from git, you can use `git rm --cached xxxx` to remove it.